### ISSUE TITLE
direct rec without det

### ISSUE THREAD

**Issue by Alice-Thirty** [JaidedAI/EasyOCR#782]
![image](https://user-images.githubusercontent.com/108393685/178737453-15683b2a-dd0b-467c-8a9a-4a4a8ca46227.png)
If I input the image to rec without detection, is it good?

I dont do the detection, like this
![image](https://user-images.githubusercontent.com/108393685/178737926-7057cdfb-811c-47f2-b0d7-497c4764e158.png)

**Comment by rkcosmos** [comment]
yes, you need to input 2 boxes if you don't use `detect` method.
